src/plugins/plan/components/PlanView.vue

```diff
@@ -143,13 +142,15 @@ export default {
     this.canvasContext = canvas.getContext('2d');
     this.setDimensions();
     this.setTimeContext();
-    this.resizeTimer = setInterval(this.resize, RESIZE_POLL_INTERVAL);
     this.handleConfigurationChange(this.configuration);
     this.planViewConfiguration.on('change', this.handleConfigurationChange);
     this.loadComposition();
+
+    this.resizeObserver = new ResizeObserver(this.resize);
+    this.resizeObserver.observe(this.$refs.plan);
   },
   beforeUnmount() {
-    clearInterval(this.resizeTimer);
+    this.resizeObserver.disconnect();
```

Comment:
we should use our [ResizeObserver composable](https://github.com/nasa/openmct/blob/43cc96332856c680ab3d8b9db8877a35628a19d0/src/ui/composables/resize.js#L34) here

Reply:
We have a composable? Will fix

Reply:
Actually I will maybe tackle this with the e2e test pr.